build-targets.js

```diff
@@ -1,4 +1,12 @@
-function buildTargets (currentNodeAbi, supportedTargets) {
+function buildTargets (currentNodeAbi, supportedTargets, pkg) {
+  // Build only for the napi runtime when node-addon-api and napi_versions are found
+  if (pkg.dependencies && pkg.dependencies['node-addon-api'] && pkg.binary && pkg.binary.napi_versions) {
+    return pkg.binary.napi_versions.map(napiVersion => ({
+      runtime: 'napi',
```

Comment:
Using `napi` as a runtime value feels wrong, are we doing that anywhere else? Sometimes you'll still need a separate node and electron build.

Reply:
Yes, `napi` is a value accepted by prebuild's `--runtime` flag  - see https://github.com/prebuild/prebuild#help

Once the switch to `napi` has been made, you'll no longer need (or be able) to build for `node` and `electron` runtimes (I was initially confused by this also).

Reply:
> Yes, napi is a value accepted by prebuild's --runtime flag

OK, good to know.

> you'll no longer need (or be able) to build for node and electron runtimes (I was initially confused by this also).

This isn't always true (it should be, but isn't). Builds that targeted node don't work in Electron 3 on Windows, for example. Hopefully that was a one-time occurrence, but I don't know if we should outright remove the ability to build explicitly for node or electron.

Reply:
It's a discussion for another day though, seeing as `runtime: napi` is already a thing.

Reply:
Is https://github.com/nodejs/node-addon-api/issues/269 the Electron 3 / Windows problem? If so, that's definitely a bit annoying but at least it looks like a delay-load-hook could be added to node-addon-api to fix it. I'll change this PR so we continue to build for electron and node runtimes also. I guess https://github.com/prebuild/prebuild/issues/206 will allow more granular control over this in the future anyway. Thanks for the quick review!